pkg/parser/uv_graph.go

```diff
@@ -71,9 +72,9 @@ func parseUvPackageLockAsGraph(lockfilePath string, config *ParserConfig) (*mode
 
 	defer func() {
 		for _, pkg := range parsedLockFile.Packages {
-			if len(pkg.Metadata.RequiresDist) != 0 {
-				for _, dep := range pkg.Metadata.RequiresDist {
-					node := uvGraphFindByVersionRange(dependencyGraph, dep.Name, dep.Specifier)
```

Comment:
i think `uvGraphFindByVersionRange` is no longer needed ? please remove definition fot this too